projects/xstar/index.js

```diff
@@ -0,0 +1,12 @@
+const { sumTokens2, } = require('../helper/unwrapLPs')
+
+async function tvl(api) {
+  
+  return sumTokens2({ owners: [
+    '0xFEc8a6cFaCbF3dA3E0c9988D59542eB6A5d49184',
```

Comment:
hi @zerokarafont what is this contract? can users withdraw funds from it?

Reply:
this is controller contract, accept native currency for income, only deployer can withdraw funds

Reply:
hmm, we cant count it as tvl since users cant withdraw funds